bin/resource_management/resource-broken-links-csv.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import pandas as pd
+import urllib
+import sys
+from sys import stderr, stdout, argv
+from multiprocessing import Process
+
+
+# In[ ]:
+
+
+def run_cpu_tasks_in_parallel(tasks):
+    """
+    Manages processes
+    tasks - list of methods to be run
+    """
+    running_tasks = [Process(target=task) for task in tasks]
+    for running_task in running_tasks:
+        running_task.start()
+    for running_task in running_tasks:
+        running_task.join()
+        
+def main(filename,processes=4):
```

Comment:
these jobs are going to be network bound not CPU bound, I bet you can run a lot more than 4 at a time.